jaxrs-tck/src/main/java/jakarta/ws/rs/tck/ee/rs/container/requestcontext/security/JAXRSClientIT.java

```diff
@@ -68,15 +69,15 @@ void logFinishTest(TestInfo testInfo) {
     TestUtil.logMsg("FINISHED TEST : "+testInfo.getDisplayName());
   }
 
-  @Deployment(testable = false)
+  @Deployment(testable = true)
   public static WebArchive createDeployment() throws IOException {
 
     InputStream inStream = JAXRSClientIT.class.getClassLoader().getResourceAsStream("jakarta/ws/rs/tck/ee/rs/container/requestcontext/security/web.xml.template");
     String webXml = editWebXmlString(inStream);
 
     WebArchive archive = ShrinkWrap.create(WebArchive.class, "jaxrs_ee_rs_container_requestcontext_security_web.war");
     archive.addClasses(TSAppConfig.class, Resource.class, RequestFilter.class);
-    archive.addAsWebInfResource("jakarta/ws/rs/tck/ee/rs/container/requestcontext/security/jaxrs_ee_rs_container_requestcontext_security_web.war.sun-web.xml", "sun-web.xml");
+//    archive.addAsWebInfResource("jakarta/ws/rs/tck/ee/rs/container/requestcontext/security/jaxrs_ee_rs_container_requestcontext_security_web.war.sun-web.xml", "sun-web.xml");
```

Comment:
maybe just remove instead of commenting out?  or maybe we could replace with some sort of generic comment like "make sure your test runner packages a vendor-specific deployment descriptor that includes the security users/passwords/roles/etc."?

Reply:
How/where do we document that this test (and others) needs something equivalent to `archive.addAsWebInfResource("jakarta/ws/rs/tck/ee/rs/container/requestcontext/security/jaxrs_ee_rs_container_requestcontext_security_web.war.sun-web.xml", "sun-web.xml");`?

Reply:
I meant to keep the commented code for a vendor to understand what's missing in the war. While it is better to have it documented somewhere, the commented row works as a commentary for people who like to visit the code. So I'd better to keep it there, but I can remove it if re-requested